install/wavefront-operator/README.md

```diff
@@ -22,6 +22,27 @@ To install the chart with a release name `test`:
 helm install --name test ./wavefront-operator --set wavefront.url=https://YOUR_CLUSTER.wavefront.com,wavefront.token=YOUR_API_TOKEN,clusterName=YOUR_CLUSTER_NAME --namespace test-namespace
 ```
 
+Issues using helm:
+
+- CRD already exists:
+```
+Error: customresourcedefinitions.apiextensions.k8s.io <"wavefrontcollectors.wavefront.com"> already exists
+```
+
+then try running the helm command with "--no-crd-hook" flag.
+
+```
+helm install --name test ./wavefront-operator --set wavefront.url=https://YOUR_CLUSTER.wavefront.com,wavefront.token=YOUR_API_TOKEN,clusterName=YOUR_CLUSTER_NAME --namespace test-namespace --no-crd-hook
+```
+
+- Deploying multiple CR instances in same namespace: (Usually applicable for WavefrontProxy CR)
```

Comment:
Let's remove this part about deploying multiple proxies in the same namespace. It's a corner case that is not that likely. They can always deploy the proxy in a different namespace.